package.json

```diff
@@ -66,9 +66,7 @@
   },
   "homepage": "https://github.com/40818419/react-code-input#readme",
   "dependencies": {
-    "classnames": "^2.2.5",
-    "react": "^16.3.2",
-    "react-dom": "^16.3.2"
```

Comment:
These should still be put into `devDependencies` to show they are directly used by this package, relying on other packages to install your dependencies isn't as good and when using Yarn PnP won't even work as you must declare explicitly all dependencies you import in your package.